molgenis/ds-tidyverse
# Feat: rename

## To test:
- Test dsTidyverse and dsTidyverse client at the same time
- Install dsTidyverse on server of choice using `install_github("molgenis/ds-tidyverse", ref = "feat/rename")`
- Install dsTidyverseClient locally using `install_github("molgenis/ds-tidyverse", ref = "feat/rename-client")`
- It is better to install rather than using `devtools::load_all` as I've had some issues passing the checks doing it this way.
- Run `devtools::check()`
- Code review
- Run this code: 

```
ds.rename(
  df.name = "mtcars",
  tidy_select = list(new_name_1 = mpg, new_name_2 = drat),
  newobj = "test",
  datasources = conns)
```
